docs/eloquent-models/schema-builder.txt

```diff
@@ -117,6 +112,63 @@ To learn more about Laravel migrations, see
 `Database: Migrations <https://laravel.com/docs/{+laravel-docs-version+}/migrations>`__
 in the Laravel documentation.
 
+.. _laravel-eloquent-schema-validation:
+
+Implement Schema Validation
+---------------------------
+
+You can use the ``jsonSchema()`` method to implement :manual:`schema
+validation </core/schema-validation/>` when using the following schema
+builder methods:
+
+- ``Schema::create()``: When creating a new collection
+- ``Schema::table()``: When updating collection properties
+
+After you implement schema validation, the server allows you to run only
+those write operations which follow the validation rules. Use schema
+validation to restrict data types and value ranges of document fields in
+a specified collection.
+
+Before creating a collection with schema validation rules, you must
+define a JSON schema.
+
+The schema is a JSON object that contains key-value pairs
+specifying the validation rules for your collection. At the top level,
+this object must include a ``$jsonSchema`` object. The ``$jsonSchema``
```

Comment:
`$jsonSchema` doesn't need to be included in the schema argument (see the example). Since the method is `Blueprint::jsonSchema`, the `'$jsonSchema'` operation is already implied. 

So this documentation may be valid for MongoDB server but not for Laravel usage

Reply:
Oh also! Since this is PHP, the schema argument is not an object, but an associative array.